data/categories_cuisines.txt

```diff
@@ -140,6 +140,7 @@ sk:Hamburger
 
 cuisine-bubble_tea
 en:Bubble Tea|boba tea
+fr: Thé aux perles
```

Comment:
```suggestion
fr:Thé aux perles
```
@vng will the search work properly with such translation/keyword?

Reply:
Hm, good case to check when 'the' is a stopword in En, but meaningful in Fr.